src/mocks/coupons.ts

```diff
@@ -0,0 +1,45 @@
+import { Coupon } from '../types/Coupon.type';
+
+export const fixedCoupon = {
```

Comment:
여기에서 타입을 지정해주면 좋을거 같아요
```suggestion
export const fixedCoupon: Coupon = {
```

Reply:
이렇게 사용하면 타입 단언을 제거하고도 타입을 명시해 줄 수 있겠네요 🙌🏻 피드백 반영해 보았습니다 🤗